first-analysis-steps/code/analysis-productions/info.yaml

```diff
@@ -0,0 +1,17 @@
+defaults:
+    application: DaVinci/v45r5
+    wg: Charm
+    automatically_configure: yes
```

Comment:
It's not required but it's probably worth including so people realise it's required for turbo data:
```suggestion
    automatically_configure: yes
    turbo: no
```
I don't think it's possible to avoid having a `turbo` option as it's ambiguous for MC. (@alexpearce?)

Reply:
Indeed! Most MC samples have Turbo and Stripping output in them, and one has to choose one to study in a give run of DaVinci.

Reply:
Testing this locally, `turbo` doesn't seem to be a valid option:
```
strictyaml.exceptions.YAMLValidationError: while parsing a mapping
unexpected key not in schema 'turbo'
  in "<unicode string>", line 5, column 1:
      turbo: no
    ^ (line: 5)
```

Reply:
Yeah that's a bug that Shantam found earlier today. I'll fix it tomorrow.